Hexastore.Test/TripleTest.cs

```diff
@@ -112,7 +112,7 @@ public void GetBy_SP_Returns()
         public void GetBy_P_Returns()
         {
             _set.Assert("s1", "p1", "o1");
-            _set.Assert("s1", "p1", TripleObject.FromData("o1"));
+            _set.Assert("s1", "p1", TripleObject.FromData("o2", true));
```

Comment:
FromData is @value representation. This should stay false.

Reply:
done